documentation/configuration.md

```diff
@@ -577,13 +834,28 @@ The `IncludePaths` and `ExcludePaths` support [wildcards and globbing](#aspnet-r
 
 Usage that is satisfied when request count is higher than 500 requests during a 1 minute period for all paths under the `/api` route:
 
-```json
-{
-  "RequestCount": 500,
-  "SlidingWindowDuration": "00:01:00",
-  "IncludePaths": [ "/api/**/*" ]
-}
-```
+<details>
+  <summary>JSON</summary>
+
+  ```json
+  {
+    "RequestCount": 500,
+    "SlidingWindowDuration": "00:01:00",
+    "IncludePaths": [ "/api/**/*" ]
+  }
+  ```
+</details>
+
+<details>
+  <summary>Environment Variables</summary>
+  
+  ```bash
+  export DotnetMonitor_CollectionRules__RuleName__Trigger__Type="AspNetRequestCount"
```

Comment:
The json example doesn't have this since it is only demonstrating the settings of the trigger.

Reply:
Same applies to all of the other trigger and action settings examples.

Reply:
Yep, I figured this would come up - the reason I added it was for copy-pastability, so people wouldn't just add the settings lines and assume it should work, even though we don't have it in the json. That being said, if you'd rather not have it for the sake of consistency, I'll yank them all out

Reply:
People already can't just take the json and copy/paste it as-is. Let's remove them to make them consistent.